Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesD.swift

```diff
@@ -3495,7 +3503,7 @@ public struct DoStmtSyntax: StmtSyntaxProtocol, SyntaxHashable, _LeafStmtSyntaxN
     }
   }
   
-  public var unexpectedBetweenDoKeywordAndBody: UnexpectedNodesSyntax? {
+  public var unexpectedBetweenDoKeywordAndThrowsClause: UnexpectedNodesSyntax? {
```

Comment:
Related to https://github.com/apple/swift-syntax/pull/2379#discussion_r1414399964, this needs a compatibility layer or a mention in the release notes.